{"uid":"e9ec09a5f05cf4ea","name":"Card Creation","fullName":"tests.regression_ui.test_ui_regression.TestUIRegression#test_card_creation","historyId":"37b04428e9c82a14933f1043cf15ed79","time":{"start":1733889843558,"stop":1733889881879,"duration":38321},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting","statusTrace":"self = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f26d6f74a60>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"afdc58d4e2e89b798a571c6d0382c214\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC004\n    @allure.title('Card Creation')\n    def test_card_creation(self, driver):\n        time.sleep(10)\n        with allure.step('Log in to Trello account.'):\n    \n            self.test_login(driver)\n        with allure.step('Open a board with an existing list and create a new card'):\n            try:\n                WebDriverWait(driver, 10).until(EC.element_to_be_clickable((By.XPATH, self.SCC.Board.BOARD_TITLE))).click()\n>               WebDriverWait(driver, 10).until(EC.visibility_of_element_located((By.XPATH, self.SCC.Board.LIST_TITLE)))\n\ntests/regression_ui/test_ui_regression.py:133: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"afdc58d4e2e89b798a571c6d0382c214\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x7f26d6da1af0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x561e534de31a <unknown>\nE       #1 0x561e52ff46e0 <unknown>\nE       #2 0x561e530433e6 <unknown>\nE       #3 0x561e53043681 <unknown>\nE       #4 0x561e53088b04 <unknown>\nE       #5 0x561e5306748d <unknown>\nE       #6 0x561e53085ed7 <unknown>\nE       #7 0x561e53067203 <unknown>\nE       #8 0x561e53035cc0 <unknown>\nE       #9 0x561e53036c9e <unknown>\nE       #10 0x561e534abd0b <unknown>\nE       #11 0x561e534afc92 <unknown>\nE       #12 0x561e53498b3c <unknown>\nE       #13 0x561e534b0807 <unknown>\nE       #14 0x561e5347e0df <unknown>\nE       #15 0x561e534cd578 <unknown>\nE       #16 0x561e534cd740 <unknown>\nE       #17 0x561e534dd196 <unknown>\nE       #18 0x7fc951c94ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f26d6f74a60>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"afdc58d4e2e89b798a571c6d0382c214\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC004\n    @allure.title('Card Creation')\n    def test_card_creation(self, driver):\n        time.sleep(10)\n        with allure.step('Log in to Trello account.'):\n    \n            self.test_login(driver)\n        with allure.step('Open a board with an existing list and create a new card'):\n            try:\n                WebDriverWait(driver, 10).until(EC.element_to_be_clickable((By.XPATH, self.SCC.Board.BOARD_TITLE))).click()\n                WebDriverWait(driver, 10).until(EC.visibility_of_element_located((By.XPATH, self.SCC.Board.LIST_TITLE)))\n                driver.find_element(By.XPATH, self.SCC.List.ADD_A_CARD_BUTTON).click()\n                driver.find_element(By.XPATH, self.SCC.List.CARD_NAME_FIELD).click()\n                time.sleep(1)\n                driver.find_element(By.XPATH, self.SCC.List.CARD_NAME_FIELD).send_keys('new_card')\n                driver.find_element(By.XPATH, self.SCC.List.ADD_CARD_SUBMIT_BUTTOMN).click()\n            except Exception as e:\n>               log.info(\"An error occurred: \", str(e))\n\ntests/regression_ui/test_ui_regression.py:140: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:2082: in info\n    root.info(msg, *args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1446: in info\n    self._log(INFO, msg, args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1589: in _log\n    self.handle(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1599: in handle\n    self.callHandlers(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1661: in callHandlers\n    hdlr.handle(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:954: in handle\n    self.emit(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py:386: in emit\n    super().emit(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1093: in emit\n    self.handleError(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1085: in emit\n    msg = self.format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:929: in format\n    return fmt.format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py:139: in format\n    return super().format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:668: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <LogRecord: root, 20, /home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py, 140, \"An error occurred: \">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:373: TypeError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1733889843284,"stop":1733889843557,"duration":273},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"setup","time":{"start":1733889843558,"stop":1733889843558,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting","statusTrace":"self = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f26d6f74a60>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"afdc58d4e2e89b798a571c6d0382c214\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC004\n    @allure.title('Card Creation')\n    def test_card_creation(self, driver):\n        time.sleep(10)\n        with allure.step('Log in to Trello account.'):\n    \n            self.test_login(driver)\n        with allure.step('Open a board with an existing list and create a new card'):\n            try:\n                WebDriverWait(driver, 10).until(EC.element_to_be_clickable((By.XPATH, self.SCC.Board.BOARD_TITLE))).click()\n>               WebDriverWait(driver, 10).until(EC.visibility_of_element_located((By.XPATH, self.SCC.Board.LIST_TITLE)))\n\ntests/regression_ui/test_ui_regression.py:133: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"afdc58d4e2e89b798a571c6d0382c214\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x7f26d6da1af0>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x561e534de31a <unknown>\nE       #1 0x561e52ff46e0 <unknown>\nE       #2 0x561e530433e6 <unknown>\nE       #3 0x561e53043681 <unknown>\nE       #4 0x561e53088b04 <unknown>\nE       #5 0x561e5306748d <unknown>\nE       #6 0x561e53085ed7 <unknown>\nE       #7 0x561e53067203 <unknown>\nE       #8 0x561e53035cc0 <unknown>\nE       #9 0x561e53036c9e <unknown>\nE       #10 0x561e534abd0b <unknown>\nE       #11 0x561e534afc92 <unknown>\nE       #12 0x561e53498b3c <unknown>\nE       #13 0x561e534b0807 <unknown>\nE       #14 0x561e5347e0df <unknown>\nE       #15 0x561e534cd578 <unknown>\nE       #16 0x561e534cd740 <unknown>\nE       #17 0x561e534dd196 <unknown>\nE       #18 0x7fc951c94ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f26d6f74a60>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"afdc58d4e2e89b798a571c6d0382c214\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC004\n    @allure.title('Card Creation')\n    def test_card_creation(self, driver):\n        time.sleep(10)\n        with allure.step('Log in to Trello account.'):\n    \n            self.test_login(driver)\n        with allure.step('Open a board with an existing list and create a new card'):\n            try:\n                WebDriverWait(driver, 10).until(EC.element_to_be_clickable((By.XPATH, self.SCC.Board.BOARD_TITLE))).click()\n                WebDriverWait(driver, 10).until(EC.visibility_of_element_located((By.XPATH, self.SCC.Board.LIST_TITLE)))\n                driver.find_element(By.XPATH, self.SCC.List.ADD_A_CARD_BUTTON).click()\n                driver.find_element(By.XPATH, self.SCC.List.CARD_NAME_FIELD).click()\n                time.sleep(1)\n                driver.find_element(By.XPATH, self.SCC.List.CARD_NAME_FIELD).send_keys('new_card')\n                driver.find_element(By.XPATH, self.SCC.List.ADD_CARD_SUBMIT_BUTTOMN).click()\n            except Exception as e:\n>               log.info(\"An error occurred: \", str(e))\n\ntests/regression_ui/test_ui_regression.py:140: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:2082: in info\n    root.info(msg, *args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1446: in info\n    self._log(INFO, msg, args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1589: in _log\n    self.handle(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1599: in handle\n    self.callHandlers(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1661: in callHandlers\n    hdlr.handle(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:954: in handle\n    self.emit(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py:386: in emit\n    super().emit(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1093: in emit\n    self.handleError(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1085: in emit\n    msg = self.format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:929: in format\n    return fmt.format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py:139: in format\n    return super().format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:668: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <LogRecord: root, 20, /home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py, 140, \"An error occurred: \">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:373: TypeError","steps":[{"name":"Log in to Trello account.","time":{"start":1733889853568,"stop":1733889871024,"duration":17456},"status":"passed","steps":[{"name":"Navigate to the Trello login page","time":{"start":1733889853569,"stop":1733889856922,"duration":3353},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Enter valid credentials (email, password, and 6-digits verification code)","time":{"start":1733889856922,"stop":1733889871024,"duration":14102},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"stepsCount":2,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Open a board with an existing list and create a new card","time":{"start":1733889871024,"stop":1733889881827,"duration":10803},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting\n","statusTrace":"  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 140, in test_card_creation\n    log.info(\"An error occurred: \", str(e))\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 2082, in info\n    root.info(msg, *args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1446, in info\n    self._log(INFO, msg, args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1589, in _log\n    self.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1599, in handle\n    self.callHandlers(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1661, in callHandlers\n    hdlr.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 954, in handle\n    self.emit(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py\", line 386, in emit\n    super().emit(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1093, in emit\n    self.handleError(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1085, in emit\n    msg = self.format(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 929, in format\n    return fmt.format(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py\", line 139, in format\n    return super().format(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 668, in format\n    record.message = record.getMessage()\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 373, in getMessage\n    msg = msg % self.args\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"572a8c38d8d4cbd7","name":"log","source":"572a8c38d8d4cbd7.txt","type":"text/plain","size":58523}],"parameters":[],"stepsCount":4,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1733889882009,"stop":1733889882084,"duration":75},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"TC004"},{"name":"tag","value":"TC000"},{"name":"parentSuite","value":"tests.regression_ui"},{"name":"suite","value":"test_ui_regression"},{"name":"subSuite","value":"TestUIRegression"},{"name":"host","value":"fv-az881-26"},{"name":"thread","value":"3218-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.regression_ui.test_ui_regression"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":4,"skipped":0,"passed":8,"unknown":0,"total":12},"items":[{"uid":"b1375d4f74d28a48","status":"broken","statusDetails":"TypeError: not all arguments converted during string formatting","time":{"start":1733888700238,"stop":1733888738729,"duration":38491}},{"uid":"dbe0b17e2f869d8d","status":"passed","time":{"start":1733843222348,"stop":1733843255086,"duration":32738}},{"uid":"9fa7d1dc42d12fae","status":"broken","statusDetails":"TypeError: not all arguments converted during string formatting","time":{"start":1733842712495,"stop":1733842750891,"duration":38396}},{"uid":"bf395ad70de3111b","status":"broken","statusDetails":"TypeError: not all arguments converted during string formatting","time":{"start":1733841866105,"stop":1733841904338,"duration":38233}},{"uid":"427dfdead3e9b445","status":"passed","time":{"start":1733840260094,"stop":1733840292854,"duration":32760}},{"uid":"e47f88cec87f6072","status":"passed","time":{"start":1733797204126,"stop":1733797232419,"duration":28293}},{"uid":"20e98e3a890bf54a","status":"passed","time":{"start":1733586085542,"stop":1733586113888,"duration":28346}},{"uid":"2a080ef6a887df51","status":"passed","time":{"start":1733548536303,"stop":1733548564843,"duration":28540}},{"uid":"1a754479eff2754","status":"passed","time":{"start":1733548292874,"stop":1733548321397,"duration":28523}},{"uid":"c9e93e81ad7dcbbb","status":"passed","time":{"start":1733547863420,"stop":1733547891929,"duration":28509}},{"uid":"74029b78a968b6d8","status":"passed","time":{"start":1733451869996,"stop":1733451889794,"duration":19798}}]},"tags":["TC004","TC000"]},"source":"e9ec09a5f05cf4ea.json","parameterValues":[]}